src/main/java/com/conveyal/datatools/manager/DataManager.java

```diff
@@ -9,17 +9,7 @@
 import com.conveyal.datatools.editor.controllers.api.SnapshotController;
 import com.conveyal.datatools.manager.auth.Auth0Connection;
 import com.conveyal.datatools.manager.controllers.DumpController;
-import com.conveyal.datatools.manager.controllers.api.AppInfoController;
-import com.conveyal.datatools.manager.controllers.api.DeploymentController;
-import com.conveyal.datatools.manager.controllers.api.FeedSourceController;
-import com.conveyal.datatools.manager.controllers.api.FeedVersionController;
-import com.conveyal.datatools.manager.controllers.api.GtfsPlusController;
-import com.conveyal.datatools.manager.controllers.api.NoteController;
-import com.conveyal.datatools.manager.controllers.api.OrganizationController;
-import com.conveyal.datatools.manager.controllers.api.ProjectController;
-import com.conveyal.datatools.manager.controllers.api.ServerController;
-import com.conveyal.datatools.manager.controllers.api.StatusController;
-import com.conveyal.datatools.manager.controllers.api.UserController;
+import com.conveyal.datatools.manager.controllers.api.*;
```

Comment:
I think we have a preference for listing out all these dependencies one-by-one, but I don't think we have a standardized IDE formatting file. Maybe we should though.

Reply:
Yes IntelliJ did this for me on save, but I've disabled this now. Reverted in 
85261da